R/DataMart.R

```diff
@@ -148,9 +148,12 @@ close.qpostcon = function(con, ...)
                 encode = "raw",
                 body = upload_file(tmpfile)))
 
-    if (inherits(res, "try-error") || res$status_code != 200)
+    if (res$status_code == 413) {
+        stopBadRequest(res, "Could not write to Displayr Cloud Drive. Data to write is too large.")
+    }
+    else if (inherits(res, "try-error") || res$status_code != 200)
     {
-        stopBadRequest(res, "Could not write to data mart.")
+        stopBadRequest(res, "Could not write to Displayr Cloud Drive. Please try again later.")
```

Comment:
Given that we do not expect this toever happen, I think it should call `mpwarn()` to add a warning to Kibana.